apps/gitness/package.json

```diff
@@ -16,13 +16,22 @@
   "dependencies": {
     "@harnessio/canary": "workspace:*",
     "@harnessio/code-service-client": "1.2.0",
+    "@harnessio/forms": "workspace:*",
     "@harnessio/playground": "workspace:*",
+    "@harnessio/unified-pipeline": "workspace:*",
+    "@harnessio/yaml-editor": "workspace:*",
     "@tanstack/react-query": "4.36.1",
+    "clsx": "^2.1.1",
+    "monaco-editor": "0.50.0",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "restful-react": "^15.9.4",
```

Comment:
We're not going to use `restful-react` any more. Do we still need it somewhere?

Reply:
Removed